### PR TITLE
Fix inconsistent `show-names` color

### LINKED ISSUE #4847: `show-names` text color is inconsistent
### Please ensure:

- [X] The bug is caused by Refined GitHub. It doesn't happen if I disable the extension.

### Description

Notice how the regular names are gray (just like time) but not in review comments:

image

This is because color from `.review-comment` is overriding the color from `.TimelineItem-body` (P.S. `.timeline-comment-header` also uses gray color).

### Screenshot

_No response_

### Console errors

_No response_

### Example URL

https://github.com/sindresorhus/refined-github/pull/4821#discussion_r716093511

### Browser(s) used

Chrome

### PR DESCRIPTION
**Pull request by MrBartusek**
Fixes #4847

## Test URLs

https://github.com/sindresorhus/refined-github/pull/4821#discussion_r716093511

## Screenshot

Before
image

After
image

### CODE REVIEW

File: source/features/show-names.css [hunk 1]
```diff
@@ -2,3 +2,7 @@
 .timeline-comment-header-text bdo {
 	display: inline-block;
 }
+
+.review-comment h4 {
```
Comment (kidonng) [inline source/features/show-names.css:6]: No need to write a new rule, just use `.color-text-secondary`.

**Comment by kidonng** [comment]
You can change the following lines:

https://github.com/sindresorhus/refined-github/blob/1d858670461a256b29dda730112a0e965b48f6f2/source/features/show-names.tsx#L56-L64

To:

```tsx
				insertionPoint.after(
					// That obnoxious part is to comfort the linter
					<span className="color-text-secondary">{`${'('}`}
						<bdo className="css-truncate css-truncate-target" style={{maxWidth: '200px'}}>
							{names[userKey].name}
						</bdo>)
					</span>,
				);
```

File: source/features/show-names.tsx [hunk 2]
```diff
@@ -54,13 +54,12 @@ async function init(): Promise<false | void> {
 			} else {
 				const insertionPoint = usernameElement.parentElement!.tagName === 'STRONG' ? usernameElement.parentElement! : usernameElement;
 				insertionPoint.after(
-					' (',
-					<bdo className="css-truncate d-inline-block">
-						<span className="css-truncate-target" style={{maxWidth: '200px'}}>
+					// That obnoxious part is to comfort the linter
+					<span className="color-text-secondary">{`${'('}`}
```
Comment (fregante) [inline source/features/show-names.tsx:58]: Lol what a mess here, you're right. I'm gonna open an issue about this on the linter’s repo

File: source/features/show-names.tsx [hunk 3]
```diff
@@ -54,13 +54,12 @@ async function init(): Promise<false | void> {
...
+						<bdo className="css-truncate css-truncate-target" style={{maxWidth: '200px'}}>
```
Comment (fregante) [inline source/features/show-names.tsx:59]: The css-truncate class should be on the parent, or else I don't remember if it affects css-truncate-target

File: source/features/show-names.tsx [hunk 4]
```diff
@@ -46,21 +46,21 @@ async function init(): Promise<false | void> {
...
 				insertionPoint.after(
-					' (',
-					<bdo className="css-truncate d-inline-block">
-						<span className="css-truncate-target" style={{maxWidth: '200px'}}>
-							{names[userKey].name}
-						</span>
-					</bdo>,
-					') ',
+					<span className="color-text-secondary css-truncate">
+						(<bdo className="css-truncate-target" style={{maxWidth: '200px'}}>{name}</bdo>)
```
Comment (fregante) [inline source/features/show-names.tsx:62]: It'd be great to check whether the `maxWidth` and the `insertionPoint` logic are still needed as well, but I'll merge this as is for now

**Review by fregante — APPROVED** [review]

**Comment by fregante** [comment]
Dziękuję za pomoc Bartosz ☺️

File: source/features/show-names.tsx (hunk 3 above)
Comment (kidonng) [reply source/features/show-names.tsx]: That CSS rule also works if the two classes are in the same element, however I agree we should keep the original form. An oversight when I was tweaking elements and linter😃

**Comment by yakov116** [comment]
After this PR

image

**Pull request merged**